opentelemetry-appender-tracing/src/layer.rs

```diff
@@ -27,6 +25,25 @@ impl<'a> tracing::field::Visit for EventVisitor<'a> {
         }
     }
 
+    fn record_str(&mut self, field: &tracing_core::Field, value: &str) {
+        if let Some(ref mut map) = self.log_record.attributes {
+            map.insert(field.name().into(), value.to_owned().into());
+        } else {
+            let mut map: OrderMap<Key, AnyValue> = OrderMap::with_capacity(1);
+            map.insert(field.name().into(), value.to_owned().into());
```

Comment:
Aren't you forgetting to set `self.log_record.attributes` here?

Reply:
Good catch. Was testing with the right code, and then missed during the commit. Fixed now :)